.github/workflows/linter.yml

```
@@ -5,19 +5,18 @@ on:
     branches:
       - main
   push:
-    branches-ignore:
+    branches:
```

Wouldn't this would end up with the linter running twice for commits made to an existing PR branch?

```suggestion
    branches-ignore:
```

---

It will just re-run CI when the PR is eventually merged into `main`. I like this approach though because then the status badges on the README are up-to-date with what has landed on the default branch (`main` in this case).

---

Ah that's a fair point! In that case ignore my suggestion ;)